cmd/fleet/serve.go

```diff
@@ -624,6 +624,10 @@ func cronVulnerabilities(
 		level.Info(logger).Log("vulnerability scanning", "not configured")
 		return
 	}
+	if !appConfig.HostSettings.EnableSoftwareInventory {
```

Comment:
Should we warn users (or exit) if `EnableSoftwareInventory=false` but `VulnerabilitySettings.DatabasesPath` is set (i.e. vuln scanning enabled but software inventory disabled)?

Other idea: Also, why not make it so that if you have `appConfig.VulnerabilitySettings.DatabasesPath` set, then software inventory is enabled.

Reply:
We will but in 5.0.0.